src/main/Duke.java

```diff
@@ -0,0 +1,97 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+public class Duke {
+    private static boolean is_bye = false;
+    private static ArrayList<Task> tasks = new ArrayList<Task>();
+    public enum Commands {
+        bye,
+        list,
+        done,
+        deadline,
+        
+    }
+
+    public static void main(String[] args) {
```

Comment:
It might be possible to have more abstraction for the operations in this Duke class. For example, Ui could handle the printing to screen and Parser could handle more of the processing.